docs/getting_started.md

```diff
@@ -203,6 +203,167 @@ See these resources for more information on clustering:
  - [Quantifying clustering behavior in Orchestrating Single Cell Analysis](https://bioconductor.org/books/release/OSCA.advanced/clustering-redux.html#quantifying-clustering-behavior)
 
 
+### Cell type annotation
+
+Processed objects may contain cell type annotations and associated metadata from one or more of the following sources.
+
+- Submitter-provided annotations
+- Automated annotations from [`SingleR`](https://bioconductor.org/packages/release/bioc/html/SingleR.html), a reference-based method ([Looney _et al._ 2019](https://doi.org/10.1038/s41590-018-0276-y)).
+- Automated annotations from [`CellAssign`](https://github.com/Irrationone/cellassign), a marker-gene based method ([Zhang _et al._ 2019](https://doi.org/10.1038/s41592-019-0529-1)).
+
+Below we provide instructions on how to access each type of cell type annotation, if present.
+
+
+#### Submitter-provided annotations
+
+To access submitter-provided annotations in the `SingleCellExperiment`, use the following command:
+
+```r
+# submitter-provided annotations for each cell
+processed_sce$submitter_celltype_annotation
+```
+
+To access submitter-provided annotations in the `AnnData` object, use the following command:
+
+```python
+# submitter-provided annotations for each cell
+processed_adata.obs["submitter_celltype_annotation"]
+```
+
+Cells that submitters did not annotate are labeled with `NA`.
+Note that submitter-provided annotations are also present in unfiltered and filtered objects and can be accessed using the same approach shown here for processed objects.
+
+
+#### `SingleR` annotations
+
+To access automated `SingleR` annotations in the `SingleCellExperiment`, use the following command:
+
+```r
+# SingleR annotations for each cell
+processed_sce$singler_celltype_annotation
+```
+
+To access automated `SingleR` annotations in the `AnnData` object, use the following command:
+
+```python
+# SingleR annotations for each cell
+processed_adata.obs["singler_celltype_annotation"]
+```
+
+Cells that `SingleR` could not confidently annotate are labeled with `NA`.
+
+
+If `SingleR` annotation was performed using ontology reference labels (default), the cell type annotation ontology terms will also be provided.
+
+To access cell type annotation ontology terms in the `SingleCellExperiment` object, use the following command:
+```r
+# SingleR ontology annotations
+processed_sce$singler_celltype_ontology
+```
+
+To access cell type annotation ontology terms in the `AnnData` object, use the following command:
+```r
+# SingleR ontology annotations
+processed_adata.obs["singler_celltype_ontology"]
+```
+
```

Comment:
Since we are always using the ontology label for the portal, I would condense this with the previous commands. And say that we are storing both the annotations and ontology labels and then in the example code chunk include both lines of code.